godtools/App/Features/ToolsFilter/Presentation/ToolFilterCategorySelectionView/ToolFilterCategorySelectionViewModel.swift

```diff
@@ -11,7 +11,7 @@ import Combine
 
 class ToolFilterCategorySelectionViewModel: ObservableObject {
```

Comment:
Hey @rachaelblue I just noticed something else in both ToolFilterCategorySelectionViewModel.swift and ToolFilterLanguageSelectionViewModel.swift that could cause problems.  This is another one of those things in Combine we have to look out for.  It might be good to make a running list of these in the ReadMe.

The issue is when storing the selected category and language via publisher and using ```.store(in: &cancellables)``` where cancellables is a reference to the scope of the ViewModel.  Once that ViewModel is deallocated it will terminate any publishers in local cancellables.

Reply:
Thanks @levieggertcru, good point.  I added a set `staticCancellables`.  Definitely need to remember to think about scope when storing publishers.